components/app.js

```diff
@@ -32,7 +37,34 @@ class App {
       }
     );
   }
+  createGrade(name, course, grade) {
+    console.log(name, course, grade);
```

Comment:
Make sure to cleanup any `console.log`s before making a pull request to `master`. Production code shouldn't have any `console.log`s in it, the only exception being `console.error` for errors.